packages/web3-core-requestmanager/src/index.js

```diff
@@ -272,6 +275,29 @@ RequestManager.prototype.clearSubscriptions = function (keepIsSyncing) {
         this.provider.reset();
 };
 
+/**
+ * Evaluates WS close event
+ *
+ * @method _isCleanClose
+ *
+ * @returns {boolean}
+ */
+RequestManager.prototype._isCleanCloseEvent = function (event) {
+    return typeof event === 'object' &&
```

Comment:
Could you change it to have this on one line? If I'm right does it not exceed the maximum characters per line.

Reply:
... and the parameters are missing in the funcDoc as well

Reply:
I've improved the ``param`` notation of the ``_isCleanCloseEvent`` funcDoc.